### PR TITLE
Update raspibolt_72_zeus-over-tor.md

### PR DESCRIPTION
**Pull request by csh2000**
Hello, connection with Tor v3 is much easier, as you do not need Client Cookies for Orbot.

Followed Openoms guide and adapted it to Raspibolt II, https://github.com/openoms/bitcoin-tutorials/blob/master/Zap_to_RaspiBlitz_through_Tor.md

Thanks!

### CODE REVIEW

**Comment by openoms** [comment]
Please note that Zap does not work over Tor since lnd 0.9.0 unless you found otherwise: https://github.com/LN-Zap/zap-iOS/issues/333

**Comment by csh2000** [comment]
Yes, thanks for the comment.

Pasted wrong link at the beginning, this is the one for Zeus, https://github.com/openoms/bitcoin-tutorials/blob/master/Zeus_to_RaspiBlitz_through_Tor.md

**Comment by Stadicus** [comment]
Awesome, thanks. Will test in the comming days.

File: raspibolt_72_zeus-over-tor.md [hunk 1]
```diff
@@ -57,8 +43,8 @@ $ go version
 If need to install Go, run these:
 
 ```
-$ wget https://storage.googleapis.com/golang/go1.11.linux-armv6l.tar.gz
-$ sudo tar -C /usr/local -xzf go1.11.linux-armv6l.tar.gz
+$ wget https://storage.googleapis.com/golang/go1.13.linux-armv6l.tar.gz
```
Comment (lucasmoten) [inline raspibolt_72_zeus-over-tor.md:46]: Consider changing to https://golang.org/dl/go1.15.2.linux-armv6l.tar.gz.
Golang 1.13 is no longer supported (By Go team)

**Pull request closed without merging**